.github/workflows/tag-master-branch-on-merge.yaml

```diff
@@ -0,0 +1,15 @@
+name: Bump version
+on:
+  push:
+    branches:
+      - master
```

Comment:
Could we change to use a different branch like main etc

Reply:
well spotted